MidasCivil_Adapter/PrivateHelpers/MetaData.cs

```diff
@@ -0,0 +1,273 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using BH.oM.Adapters.MidasCivil;
+using BH.Engine;
+using System;
+
+namespace BH.Adapter.MidasCivil
+{
+    public partial class MidasCivilAdapter
+    {
+        private bool SetMetaData(MetaData MidasMetaData)
+        {
+            string path = m_directory + @"\TextFiles\00_MetaData";
+
+            if (!Directory.Exists(path))
+            {
+                Directory.CreateDirectory(path);
+            }
+
+            path += "\\" + "PROJINFO.txt";
+
+            if (File.Exists(path))
+            {
+                Engine.Reflection.Compute.RecordWarning("PROJINFO.txt already exists, the information has been update to latest user inputs");
```

Comment:
```suggestion
                Engine.Reflection.Compute.RecordWarning("MetaData already exists, the PROJINFO has been overwritten.");
```

I would suggest you say it has been overwritten instead of updated.

You can also write an update method here: https://github.com/BHoM/MidasCivil_Toolkit/tree/master/MidasCivil_Adapter/CRUD/Update but then you need to basically compare differences rather than just overwriting.

Reply:
Updated